Applications/Spire/Include/Spire/TimeAndSales/LocalTimeAndSalesModel.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef SPIRE_LOCAL_TIME_AND_SALES_MODEL_HPP
+#define SPIRE_LOCAL_TIME_AND_SALES_MODEL_HPP
+#include <QTimer>
+#include "Nexus/Definitions/Money.hpp"
+#include "Spire/TimeAndSales/TimeAndSales.hpp"
+#include "Spire/TimeAndSales/TimeAndSalesModel.hpp"
+
+namespace Spire {
+
+  /*
+   * Implements a local TimeAndSalesModel model that periodically produces new
+   * time and sales.
+   */
+  class LocalTimeAndSalesModel : public TimeAndSalesModel {
```

Comment:
This doesn't fit the purpose of a local model, which is fine. Rename it to DemoTimeAndSalesModel and move it into the TimeAndSalesUiTester project.

Reply:
How to test the `DemoTimeAndSalesModel`? or which project to test it? As the `SpireTester` cannot access the `TimeAndSalesUiTester` project, the `DemoTimeAndSalesModel` moved to the `TimeAndSalesUiTester` project can no longer be tested in `SpireTester`.

Reply:
You don't need to test it. It's only used for demo purposes. A proper local model would need to be tested though.